packages/forms-web-app/src/views/layouts/register-main.njk

```diff
@@ -0,0 +1,10 @@
+{% extends "layouts/main.njk" %}
```

Comment:
Is this file needed, or any issue with main.njk?

Reply:
@pk2453 [As per the ticket](https://pins-ds.atlassian.net/browse/ASB-162), we only need the service name in the register journey pages, main.njk is included in other pages as well